source/fundamentals/crud/restful-api-tutorial.txt

```diff
@@ -0,0 +1,384 @@
+:atlas:`Get Started with Atlas Guide </getting-started>`. 
+
+This tutorial uses .NET Core 8.0, but you can use any version later than .NET 6.0.
+
+Set Up Your Project
+-------------------
+
+You can create a new .NET Core project using the web application template that
+Microsoft offers. To do this, run the following commands in your terminal:
+
+.. code-block:: bash
+
+   dotnet new webapi -o MongoExample
+   cd MongoExample
+
+To add the {+driver-short+} to your project as a dependency, run the following command:
+
+.. code-block:: bash
+
+   dotnet add package MongoDB.Driver
+
+The preceding commands create a new web application project for .NET core named
+``MongoExample`` and install the latest {+driver-short+}. The template project
+includes some boilerplate files. During this tutorial, you will add to these
+files and remove some of the boilerplate code to create a RESTful API. 
+
+Design a Document Model and Database Service
+--------------------------------------------
+
+In this section, you will create and configure your MongoDB service and define
+the data model for your RESTful API.
+
+.. procedure:: Create a MongoDB Service
+   :style: connected
+
+   .. step:: Create the MongoDBSettings class
+
+      Your MongoDB service will be responisble for establishing a connection and
+      directly working with documents within MongoDB. In your project, create a
+      folder named ``Models``. In the ``Models`` folder, create a new file named
+      ``MongoDBSettings.cs``. In this file, add the following code:
+
+      .. literalinclude:: /includes/fundamentals/code-examples/restful-api-tutorial/MongoDBSettingsSetup.cs
+         :language: csharp
+         :dedent:
+
+      The preceding code defines a class named ``MongoDBSettings`` that
+      contains information about your connection, the database name, and the
+      collection name.
+
+   .. step:: Update the appsettings.json file
+
+      The data that will be stored in the class fields defined in the
+      ``MongoDBSettings`` class is found in the projects' ``appsettings.json``
+      file. Open this file and add the following:
+
+      .. code-block:: json
+         :copyable: true
+
+         {
+            "Logging": {
+               "LogLevel": {
+               "Default": "Information",
+               "Microsoft.AspNetCore": "Warning"
+               }
+            },
+         "AllowedHosts": "*",
+         "MongoDB": {
+            "ConnectionURI": "ATLAS_URI_HERE",
+            "DatabaseName": "sample_mflix",
+            "CollectionName": "playlist"
+            }
+         }
+
+      Note the ``MongoDB`` field. This tutorial uses the ``sample_mflix``
+      database and the ``playlist`` collection. Replace the ``ATLAS_URI_HERE``
+      placeholder with your MongoDB Atlas connection string. For more
+      information on how to find your connection string, see the
+      :ref:`csharp-quickstart` guide.
```

Comment:
S: simplify - also, the QS guide doesnt provide info on locating the Cxn string. Perhaps link here instead - https://www.mongodb.com/docs/atlas/tutorial/connect-to-your-cluster/

```suggestion
      This tutorial uses the ``sample_mflix``
      database and the ``playlist`` collection. Replace the ``<Atlas connection string>``
      placeholder with your MongoDB Atlas connection string.
```